storage/time-series/influxdb/storer/app/ServiceStateMgmt.js

```diff
@@ -0,0 +1,68 @@
+const {
+  ServiceStateManager: { Manager },
+} = require('@dojot/microservice-sdk');
+
+
+/**
+ * This class has responsible for the health check and graceful shutdown
+ */
+class ServiceStateMgmt {
+  /**
+     *
+     */
+  constructor() {
+    this.servername = 'influxdb-storer';
+    this.manager = new Manager([this.servername]);
```

Comment:
Will there be monitoring of the connection to kafka, similar to what was done for kafka-ws?

Reply:
Yes, I think so...